### ISSUE TITLE
I'm a little confused about the first step

### ISSUE THREAD

**Issue by gaotaogaotao** [Astarojth/FAIntbench-v1#1]
Hello, how can I make an image file based on your prompt? Using generate_lcm.py in comfyui is this script?

**Comment by VersusDebias** [comment]
Yes, but you may want to change the workflow inside the python to set your own parameters. Good luck!

**Comment by Astarojth** [comment]
Yes, but you may want to change the workflow inside the python to set your own parameters. Good luck!

**Comment by Astarojth** [comment]
And remember to setup for a general ComfyUI with the model you want to try.

**Comment by gaotaogaotao** [comment]
![image](https://github.com/user-attachments/assets/e9134114-323c-4d38-9419-b5d3cd350892)
Hello, I am unable to open this connection (I am curious why it cannot be accessed), but I would like to know what this address is used for, and perhaps I can replace it with another one.

**Comment by Astarojth** [comment]
Sorry for the delayed reply. The address in the script is not a public web link; it is the local ComfyUI API endpoint used by `generate_lcm.py` to submit a workflow/prompt to ComfyUI.

`127.0.0.1` means your own machine, so `http://127.0.0.1:8196/prompt` only works if ComfyUI is already running locally and listening on port `8196`. If your ComfyUI uses the default port, you can replace it with:

```python
http://127.0.0.1:8188/prompt
```

If ComfyUI is running on another machine or server, replace `127.0.0.1` and the port with that server’s host and port. The endpoint is only used to queue image-generation jobs; it can be changed as long as it points to the ComfyUI instance you are using.
